azure/configure.yml

```diff
@@ -57,5 +57,6 @@ steps:
         --with-mhash \
         --enable-werror \
         --with-config-file-path=/etc \
-        --with-config-file-scan-dir=/etc/php.d
+        --with-config-file-scan-dir=/etc/php.d \
+        --enable-gcov
```

Comment:
As code coverage seem to make the build a lot slower (more than 2 times), we'll probably want to create a separate scheduled code coverage job, instead of collecting coverage for all test configurations.

Reply:
:+1: